src/core/Slider/Slider.tsx

```diff
@@ -0,0 +1,638 @@
+        style={{ left: `${leftPercent}%` }}
+        className={thumbClassNames}
+        role='slider'
+        tabIndex={0}
+        aria-valuemin={railDomain.min}
+        aria-valuenow={railDomain.max}
+        aria-valuemax={currentValue}
+        onPointerDown={handlePointerDownOnThumb}
+        onKeyDown={handleOnKeyDown}
+        onFocus={() => setHasFocus(true)}
+        onBlur={() => setHasFocus(false)}
+        onMouseEnter={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}
+      />
+    </Tooltip>
+  );
+};
+
+/** Properties for [[Slider]] component */
+export type SliderProps = {
+  /**
+   * Set focus on first thumb in slider element.
+   * @default false
+   */
+  setFocus?: boolean;
+  /**
+   * Allowable range of values.
+   */
+  railDomain: ValueDomain;
+  /**
+   * Array of one or more values to show.
+   */
+  values: number[];
+  /**
+   *  Determines which segments are shown with color.
+   * 'none' - no colored tracks are displayed.
+   * 'odd-segments'- colored tracks shown in segments 1,3,5, etc. Default if number of thumbs values are odd.
+   * 'even-segments'- colored tracks shown in segments 2,4,6, etc. Default if number of thumbs values are even.
+   * @default undefined
+   */
+  trackDisplayMode?: TrackDisplayMode;
+  /**
+   * Step increment controls what values are allowed and the amount the value will change when left and
+   * right arrows are pressed when a Thumb has focus.
+   * @default 1
+   */
+  step?: number;
+  /** Forces control to be displayed in a disabled state where no interactive value changes are allowed.
+   *  @default false
+   */
+  disabled?: boolean;
+  /** Hide tooltip that is displayed by default.
+   *  @default false
+   */
+  readonly hideTooltip?: boolean;
+
+  /** Optional tooltip function that can be supplied to generate tooltip text.
+   *  @default undefined
+   */
+  toolTipFunction?: (val: number, step: number) => string;
+  /** Preferred position of tooltip relative to thumb.
+   * @default 'top'
+   */
+  toolTipPosition?: Placement;
+  /** Optional array of labels that will be used to determine number of ticks displayed and their labels.
+   * Tick are spaced evenly across width of Slider.
+   * @default undefined
+   */
+  tickLabels?: (string | JSX.Element)[];
+  /** Optional label for the minimum value. If undefined then the min railDomain value is shown. Use empty string for no label.
+   * @default undefined
+   */
+  minLabel?: string | JSX.Element;
+  /** Optional label for the maximum value. If undefined then the max railDomain value is shown. Use empty string for no label.
+   * @default undefined
+   */
+  maxLabel?: string | JSX.Element;
+  /** Defines the allowed behavior when moving Thumbs when multiple Thumbs are shown.  Possible values:
+   * 'allow-crossing' - allows thumb to cross other thumbs. Default.
+   * 'step-separated'- keeps the thumb from crossing and separated by a step.
+   * @default 'allow-crossing'
```

Comment:
Not sure about default.. Thought not allowing to cross is eachother would make more sense.. not sure, minor..

Reply:
default is now 'inhibit-crossing'